packages/spark/sodasql/dialects/spark_dialect.py

```diff
@@ -57,6 +57,9 @@ def sql_tables_metadata_query(self, limit: Optional[int] = None, filter: str = N
 
     def sql_tables_metadata(self, limit: str = 10, filter: str = None):
         # TODO Implement limit
+        if self.database is None:
+            raise NotImplementedError("Can not query for tables when database is not given.")
```

Comment:
Just a small typo here, can not => cannot / can't

Reply:
Fixed